batchlib/analysis/cell_level_analysis.py

```diff
@@ -290,4 +293,7 @@ def run(self, input_files, output_files, gpu_id=None):
             _save_all_stats = partial(self.save_all_stats, device=device)
             for input_file, output_file in tqdm(list(zip(input_files, output_files)),
                                                 desc='running cell-level analysis on images'):
+                if self.outlier_predicate(input_file):
```

Comment:
I think @imagirom should double-check this and determine what's the right way to skip images in the analysis.

Reply:
This should work fine, if the goal is to simply not run the analysis on the outliers.

@constantinpape could anything go wrong in `check_output` if the corresponding result file is missing, or is that all taken care of?

Reply:
Yes, that's indeed a problem; `check_output` will fail in this case.
And in general, I don't think just skipping it is a good solution, because it's hard to see whether this was an issue in the code or because of an outlier.
I think it would be better to write the output with `None` or something similar.
(Also in the table this should be marked somehow.)

Reply:
Ok, then I would actually prefer to do that in the summary job, as in `cell_level_analysis.py` everything is per-image